src/main/java/org/folio/roles/integration/kafka/KafkaMessageListener.java

```diff
@@ -31,8 +29,10 @@ public class KafkaMessageListener {
     groupId = "#{folioKafkaProperties.listener['capability'].groupId}",
     topicPattern = "#{folioKafkaProperties.listener['capability'].topicPattern}")
   public void handleCapabilityEvent(ResourceEvent resourceEvent) {
-    try (var ignored = new FolioExecutionContextSetter(metadata, Map.of(TENANT, List.of(resourceEvent.getTenant())))) {
-      capabilityKafkaEventHandler.handleEvent(resourceEvent);
+    try (
+      var ignored = new FolioExecutionContextSetter(executionContextBuilder.buildContext(resourceEvent.getTenant()))) {
```

Comment:
It looks like the system user should be defined in MD. So, that place can be called as `executeSystemUserScoped`

Reply:
What system user?

Reply:
Done